fonts/extract-glyphs.py

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env python3
+
+import os
+from lxml import etree
+
+# Define svg namespace
+svg_ns = "http://www.w3.org/2000/svg"
+
+######################
+#  Helper Functions  #
+######################
+
+# Open file in same directory as script and retrieve its content
+def get_file_content(filepath):
+   __location__ = os.path.realpath(os.path.dirname(__file__))
+   file = open(os.path.join(__location__, filepath), 'r')
+   content = file.read()
```

Comment:
Is there any advantage to having this as a separate function, rather than just regularly opening the file? With context managers you don't need to worry about forgetting to close it anymore:

```
with open(filename, 'r') as myfile:
     # do something with myfile
# file is closed here.
```